cpp/tests/reductions/scan_tests.cpp

```diff
@@ -625,12 +626,12 @@ TEST_F(StructScanTest, StructScanMinMaxWithNulls)
   auto const input = [] {
     auto child1 = STRINGS_CW{{"año",
                               "bit",
-                              "₹1" /*null*/,
+                              "" /*null*/,
                               "aaa" /*NULL*/,
```

Comment:
Can we make these match case? null/NULL?

Reply:
Wow, sorry for missing that. I would prefer `/* parent null */` and `/* child null */`, but that's not a blocker. I'll approve this.

Reply:
I agree. There are not too many of these. I can change this as part of this PR.